### PR TITLE
Implemented selectable armor divisor in the apply damage dialog

### PR DESCRIPTION
**Pull request by Exxar**
@mjeffw we talked about this a loooong time ago. Appears to be working, except the 100 and infinite levels of armor divisor break the layout.

### CODE REVIEW

File: templates/apply-damage/apply-damage-dialog.html [hunk 1]
```diff
@@ -171,11 +171,28 @@ <h4>{{localize "GURPS.addTacticalRules"}}</h4>
...
+            <label for='tactical-armordivisor' class='label-toggle'>{{localize "GURPS.addArmorDivisor"}}: <span
+                    class='pdflink'>B378</span></label>
+            <div class='grid grid-5col continuation collapsible-content'>
+              <label><input type='radio' name='tactical-armordivisor' value='2' {{#if (eq CALC.armorDivisor 2)}}
+                            checked{{/if}}>&nbsp;2</label>
+              <label><input type='radio' name='tactical-armordivisor' value='3' {{#if (eq CALC.armorDivisor 3)}}
+                            checked{{/if}}>&nbsp;3</label>
+              <label><input type='radio' name='tactical-armordivisor' value='5' {{#if (eq CALC.armorDivisor 5)}}
+                            checked{{/if}}>&nbsp;5</label>
+              <label><input type='radio' name='tactical-armordivisor' value='100' {{#if (eq CALC.armorDivisor 100)}}
```
Comment (mjeffw) [inline templates/apply-damage/apply-damage-dialog.html]: Shouldn't this value be 10? (from B102).
```suggestion
              <label><input type='radio' name='tactical-armordivisor' value='10' {{#if (eq CALC.armorDivisor 10)}}
```

File: templates/apply-damage/apply-damage-dialog.html [hunk 2]
```diff
@@ -171,11 +171,28 @@ <h4>{{localize "GURPS.addTacticalRules"}}</h4>
...
+                            checked{{/if}}>&nbsp;100</label>
+              <label><input type='radio' name='tactical-armordivisor' value='-1' {{#if (eq CALC.armorDivisor -1)}}
```
Comment (mjeffw) [inline templates/apply-damage/apply-damage-dialog.html:190]: Your PR mentions 'infinite'. Is that what value -1 is equivalent to? Where is an 'infinite' armor divisor defined?

**Review by mjeffw — CHANGES_REQUESTED** [review]
See my two comments on AD 100 and -1.

**Comment by Exxar** [comment]
Umm, `damagecalculator.js` line 24 defines 
```
// -1 means 'Ignores DR'
const armorDivisorSteps = [-1, 100, 5, 3, 2, 1]
```
This is the division progression the existing DR hardening handling code uses. So that's where -1 as infinite comes from. I also noticed that divisor 10 is missing, but I didn't want to do anything except to introduce this new feature initially. If you're otherwise fine with this PR I'll gladly extend this to include armor divisor 10.

**Pull request merged**